copy/observatory/scoring.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Scoring Methodology
```

Comment:
As mentioned above, this page is titled "Scoring and Grading" in the sidebar.

Reply:
It has been renamed in the sidebar